flintrock/scripts/install-spark.sh

```diff
@@ -2,22 +2,26 @@
 
 set -e
 
-spark_version="$1"
+version="$1"
 distribution="$2"
+download_source="$3"
+
+url=$(eval "echo \"$download_source\"")
```

Comment:
I think doing the variable substitution in Python should eliminate code smells like this one.